klippa/src/serialize.rs

```diff
@@ -0,0 +1,394 @@
+//! serializer
+use fnv::FnvHashMap;
+use write_fonts::types::Scalar;
+
+#[derive(Clone, Copy, Debug)]
+#[allow(dead_code)]
+pub(crate) struct SerializeErrorFlags(u16);
+
+#[allow(dead_code)]
+impl SerializeErrorFlags {
+    pub const SERIALIZE_ERROR_NONE: Self = Self(0x0000);
```

Comment:
Unless there's something fancy here I think this should just be a rust enum?

Reply:
We'll need |= operator on this flag, so I'm keep it as a unsigned number